$OpenBSD$

stc++ has isfinite and signbite. Both of them are defined inline, not as macro.

--- third_party/WebKit/Source/wtf/MathExtras.h.orig.port	Wed Apr  2 15:08:53 2014
+++ third_party/WebKit/Source/wtf/MathExtras.h	Fri Apr 18 02:53:30 2014
@@ -58,7 +58,7 @@ inline double wtf_ceil(double x) { return copysign(cei
 
 #endif
 
-#if OS(OPENBSD)
+#if OS(OPENBSD) && !defined(__Bitrig__)
 
 namespace std {
 
